asv.conf.json

```diff
@@ -1,32 +1,65 @@
 {
+    "version": 1,
+
     // The versions of Python to test against
-    "pythons": ["3.6"],
+    "pythons": ["3.8"], // Python 3.7 included below with extra cached_property dependency
 
-    "environment_type": "conda",
-    "conda_channels": ["conda-forge", "defaults"],
+    // "environment_type": "virtualenv",
+    "environment_type": "virtualenv",
 
     // The matrix of dependencies to test.  Each key is the name of a
     // package (in PyPI) and the values are version numbers.  An empty
-    // list indicates to just test against the default (latest)
+    // string indicates to just test against the default (latest)
     // version.
     "matrix": {
-        "numpy": ["1.14"],
-        "numba": ["0.39"]
+        "numpy": "",
+        "numba": "",
+        "astropy": "",
+        "astroquery": "",
+        "jplephem": "",
+        "pandas": "",
+        "plotly": "",
+        "scipy": "",
+        "matplotlib": "",
     },
 
+    "include": [
```

Comment:
xref: https://asv.readthedocs.io/en/stable/asv.conf.json.html#include

Reply:
This is to make asv create only two environments for benchmarking:
- py3.8 with dependencies
- py3.7 with dependencies and cached_property